pkg/provider/provider.go

```diff
@@ -142,14 +146,41 @@ var (
 	loaded = false
 )
 
-func buildTestEnvironment() { //nolint:funlen
+// Build image with version based on environment variables if provided, else use a default value
+func buildImageWithVersion() string {
+	tnfPartnerRepo := os.Getenv("TNF_PARTNER_REPO")
```

Comment:
Parsing/reading config env vars is done through the [pkg/configuration/utils package](https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/configuration/utils.go) `init()` function, which relies on `github.com/kelseyhightower/envconfig` library to do it. I never liked it, but, to keep consistency with the other ones, you should add those env vars [TestParameters struct](https://github.com/test-network-function/cnf-certification-test/blob/a5c7e9c20fb2d23efb8d3682804beea573ec5f05/pkg/configuration/configuration.go#L125).

Reply:
Okay!  Makes sense.  This is just a copy paste from another file in our code.  Can we address with a subsequent PR?

Reply:
Of course 👍